charts/fybrik/files/taxonomy/fybrik_application.json

```diff
@@ -41,11 +41,25 @@
       "description": "DataRequirements structure contains a list of requirements (interface, need to catalog the dataset, etc.)",
       "type": "object",
       "properties": {
+        "flowParams": {
+          "$ref": "#/definitions/FlowRequirements",
+          "description": "FlowParams include the requirements for particular data flows"
+        },
         "interface": {
           "$ref": "taxonomy.json#/definitions/Interface",
           "description": "Interface indicates the protocol and format expected by the data user"
         }
       }
+    },
+    "FlowRequirements": {
```

Comment:
How was it generated? FlowRequirements has more fields than just metadata

Reply:
hmmm I ran `make all`... strange